gnovm/cmd/gno/test.go

```diff
@@ -267,14 +266,16 @@ func gnoTestPkg(
 
 	// testing with *_test.gno
 	if len(unittestFiles) > 0 {
-		memPkg := gno.ReadMemPackage(pkgPath, pkgPath)
+		// FIXME: read gnoPkgPath from gno.mod when ready
+		gnoPkgPath := strings.TrimPrefix(pkgPath, "./examples/")
```

Comment:
there should be some central README file (a spec) somewhere for how to create and understand testfiles.

does this line warrant some explanation in that README file/spec? it would say something about the examples folder.

Reply:
if that spec doesn't exist this is a good time to start one.

Reply:
Good idea, I was thinking, maybe we should also put that information in the output for `gno test --help` ?

Reply:
I suggest this approach:
- if gno.mod -> use gno.mod
- else, if `package name // import "gno.land/r/blah"` -> take from the import (similar to how Go was doing it before go.mod)
- else, generate a random name, like `r/anon/123456`

Reply:
As discussed, I removed that part in favor of using `gno.mod`. I updated the PR body accordingly : 

![image](https://github.com/gnolang/gno/assets/92280/05575e9c-2586-4fc6-b1d1-c1f8c208b609)

Reply:
@jaekwon Added a long `gno test -h` output here #1010